src/steps/account/converter.ts

```diff
@@ -5,21 +5,15 @@ import {
 
 import { Entities } from '../constants';
 
-export function createAccountEntity(): Entity {
+export function createAccountEntity(account: { hostname: string }): Entity {
   return createIntegrationEntity({
     entityData: {
-      source: {
-        id: 'acme-unique-account-id',
-        name: 'Example Co. Acme Account',
-      },
+      source: account,
       assign: {
-        _key: 'acme-unique-account-id',
+        _key: account.hostname,
         _type: Entities.ACCOUNT._type,
         _class: Entities.ACCOUNT._class,
-        mfaEnabled: true,
-        // This is a custom property that is not a part of the data model class
-        // hierarchy. See: https://github.com/JupiterOne/data-model/blob/master/src/schemas/Account.json
-        manager: 'Manager Name',
+        name: account.hostname,
```

Comment:
I'd add:

```suggestion
        webLink: ${the value of this.config.hostname},
```

The `webLink` is a "special" property that we should add whenever possible that should contain full link. This allows customers reviewing the graph to select some resource and directly click on it to get to that specific resource view UI page.